runbot_gitlab_minimal/views/runbot_repo.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_repo_form" model="ir.ui.view">
+            <field name="model">runbot.repo</field>
+            <field name="inherit_id" ref="runbot.view_repo_form"/>
+            <field name="arch" type="xml">
+                <field name="token" position="attributes">
+                    <attribute name="string">Token</attribute>
+                    <attribute name="attrs">{'required': [('uses_gitlab', '=', True)]}</attribute>
```

Comment:
We don't need a `token` required since that you are using directly a `git fetch`
This item is optional if you want use gitlab api but not required. I think so